brave-opentracing/src/main/java/io/opentracing/BraveTracerImpl.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2016 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing;
+
+import java.util.Map;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.IdConversion;
+import com.github.kristofa.brave.SpanId;
+import com.github.kristofa.brave.http.BraveHttpHeaders;
+
+import io.opentracing.propagation.Format;
+import java.util.HashMap;
+
+public final class BraveTracerImpl extends AbstractTracer {
```

Comment:
This is the only public type (I think) it would be good to put a code example here and also in the README, rather than defer completely to the opentracing website. You can look at LocalTracer for a good example of an explained class.